webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -1,35 +1,38 @@
 import React, { useState } from 'react';
-import { Tabs, Tab, TabTitleText, Label } from '@patternfly/react-core';
-import { InfoCircleIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
 
+import SkeletonLoader from 'foremanReact/components/common/SkeletonLoader';
 import { translate as __ } from 'foremanReact/common/I18n';
 
+import AnsibleVariableOverrides from './components/AnsibleVariableOverrides';
 import './AnsibleHostDetail.scss';
 
-const AnsibleHostDetail = props => {
+const AnsibleHostDetail = ({ response, status }) => {
   // https://projects.theforeman.org/issues/32398
   const [activeTab] = useState('variables');
 
   return (
-    <Tabs activeKey={activeTab} isSecondary>
-      <Tab
-        eventKey="variables"
-        title={<TabTitleText>{__('Variables')}</TabTitleText>}
-      >
-        <div className="host-details-tab-item">
-          <div className="ansible-host-detail">
-            <Label
-              color="blue"
-              icon={<InfoCircleIcon />}
-              style={{ marginTop: '1.5rem' }}
-            >
-              Ansible Variables coming soon!
-            </Label>
+    <SkeletonLoader status={status} skeletonProps={{ count: 5 }}>
```

Comment:
I think the skeleton loader isn't needed here, we can always show the tab itself, and the AnsibleVariableOverrides component already has a skeleton while it's loading

Reply:
The page makes request for the host attributes and the skeleton loader here waits for it to resolve so that it has host id available. The loader in AnsibleVariableOverrides waits for the variable values to be fetched.